diff --git a/node_modules/create-hash/browser.js b/node_modules/create-hash/browser.js
index 01841bb..ed8dc71 100644
--- a/node_modules/create-hash/browser.js
+++ b/node_modules/create-hash/browser.js
@@ -23,6 +23,7 @@ Hash.prototype._final = function () {
 
 module.exports = function createHash (alg) {
   alg = alg.toLowerCase()
+  if (alg === 'md4') alg = 'md5';
   if (alg === 'md5') return new MD5()
   if (alg === 'rmd160' || alg === 'ripemd160') return new RIPEMD160()
 
