diff --git a/node_modules/css-loader/dist/index.js b/node_modules/css-loader/dist/index.js
index 85e12ab..31105df 100644
--- a/node_modules/css-loader/dist/index.js
+++ b/node_modules/css-loader/dist/index.js
@@ -15,6 +15,8 @@ var _schemaUtils = require("schema-utils");
 
 var _semver = require("semver");
 
+var _path = require("path");
+
 var _CssSyntaxError = _interopRequireDefault(require("./CssSyntaxError"));
 
 var _Warning = _interopRequireDefault(require("./Warning"));
@@ -166,13 +168,13 @@ async function loader(content, map, meta) {
   if (options.modules.exportOnlyLocals !== true) {
     imports.unshift({
       importName: "___CSS_LOADER_API_IMPORT___",
-      url: (0, _loaderUtils.stringifyRequest)(this, require.resolve("./runtime/api"))
+      url: (0, _loaderUtils.stringifyRequest)(this, _path.resolve(__dirname, "./runtime/api"))
     });
 
     if (options.sourceMap) {
       imports.unshift({
         importName: "___CSS_LOADER_API_SOURCEMAP_IMPORT___",
-        url: (0, _loaderUtils.stringifyRequest)(this, require.resolve("./runtime/cssWithMappingToString"))
+        url: (0, _loaderUtils.stringifyRequest)(this, _path.resolve(__dirname, "./runtime/cssWithMappingToString"))
       });
     }
   }
