diff --git a/node_modules/loader-runner/lib/loadLoader.js b/node_modules/loader-runner/lib/loadLoader.js
index 1210353..6bda08c 100644
--- a/node_modules/loader-runner/lib/loadLoader.js
+++ b/node_modules/loader-runner/lib/loadLoader.js
@@ -16,7 +16,10 @@ module.exports = function loadLoader(loader, callback) {
 		}
 	} else {
 		try {
-			var module = require(loader.path);
+			if("__webpackLoaderMap" in global)
+				var module = global.__webpackLoaderMap(loader.path)
+			else
+				var module = require(loader.path);
 		} catch(e) {
 			// it is possible for node to choke on a require if the FD descriptor
 			// limit has been reached. give it a chance to recover.
