diff --git a/node_modules/minipass-fetch/lib/body.js b/node_modules/minipass-fetch/lib/body.js
index fb7ffc3..92c0c8b 100644
--- a/node_modules/minipass-fetch/lib/body.js
+++ b/node_modules/minipass-fetch/lib/body.js
@@ -133,7 +133,7 @@ class Body {
     // do not keep the process open just for this timeout, even
     // though we expect it'll get cleared eventually.
     if (resTimeout) {
-      resTimeout.unref()
+      // resTimeout.unref()
     }
 
     // do the pipe in the promise, because the pipe() can send too much
