diff --git a/node_modules/npm-registry-fetch/index.js b/node_modules/npm-registry-fetch/index.js
index df3b49e..dfd614c 100644
--- a/node_modules/npm-registry-fetch/index.js
+++ b/node_modules/npm-registry-fetch/index.js
@@ -165,7 +165,7 @@ function getCacheMode (opts) {
 
 function getHeaders (registry, uri, opts) {
   const headers = Object.assign({
-    'npm-in-ci': !!opts.isFromCI,
+    // 'npm-in-ci': !!opts.isFromCI,
     'user-agent': opts.userAgent,
   }, opts.headers || {})
 
