diff --git a/node_modules/pacote/lib/registry.js b/node_modules/pacote/lib/registry.js
index 537610d..32ed4c1 100644
--- a/node_modules/pacote/lib/registry.js
+++ b/node_modules/pacote/lib/registry.js
@@ -65,9 +65,9 @@ class RegistryFetcher extends Fetcher {
       'user-agent': this.opts.userAgent ||
         `pacote/${pacoteVersion} node/${process.version}`,
       ...(this.opts.headers || {}),
-      'pacote-version': pacoteVersion,
-      'pacote-req-type': 'packument',
-      'pacote-pkg-id': `registry:${this.spec.name}`,
+      // 'pacote-version': pacoteVersion,
+      // 'pacote-req-type': 'packument',
+      // 'pacote-pkg-id': `registry:${this.spec.name}`,
       accept: thisMetadata ? fullDoc : corgiDoc,
     }
   }
diff --git a/node_modules/pacote/lib/remote.js b/node_modules/pacote/lib/remote.js
index 91f6eb5..bbdecb5 100644
--- a/node_modules/pacote/lib/remote.js
+++ b/node_modules/pacote/lib/remote.js
@@ -51,11 +51,11 @@ class RemoteFetcher extends Fetcher {
       'user-agent': this.opts.userAgent ||
         `pacote/${pacoteVersion} node/${process.version}`,
       ...(this.opts.headers || {}),
-      'pacote-version': pacoteVersion,
-      'pacote-req-type': 'tarball',
-      'pacote-pkg-id': this.pkgid,
-      ...(this.integrity ? { 'pacote-integrity': String(this.integrity) }
-        : {}),
+      // 'pacote-version': pacoteVersion,
+      // 'pacote-req-type': 'tarball',
+      // 'pacote-pkg-id': this.pkgid,
+      // ...(this.integrity ? { 'pacote-integrity': String(this.integrity) }
+      //   : {}),
       ...(this.opts.headers || {}),
     }
   }
